example/ios/PodfileForSRTests

```diff
@@ -21,8 +21,23 @@ if linkage != nil
   use_frameworks! :linkage => linkage.to_sym
 end
 
+use_modular_headers!
```

Comment:
When running iOS SR Tests on Bitrise, I switch the Podfile with this file so we can run the tests.

`use_modular_headers!` is needed for SR tests as `React-RCTText` does not declare that it declares modules.